enable uinput via both the input group (as upstream does) and via uaccess

upstream splits this into two rules based on if systemd is available:
https://invent.kde.org/plasma/plasma-bigscreen/-/merge_requests/192
(after this release)

--- a/inputhandler/40-uinput.rules
+++ b/inputhandler/40-uinput.rules
@@ -1,3 +1,3 @@
 # SPDX-FileCopyrightText: 2026 Bart Ribbers <bribbers@disroot.org>
 # SPDX-License-Identifier: CC0-1.0
-SUBSYSTEM=="misc", KERNEL=="uinput", MODE="0660", GROUP="input"
+SUBSYSTEM=="misc", KERNEL=="uinput", MODE="0660", GROUP="input", TAG+="uaccess", OPTIONS+="static_node=uinput"
